### PR TITLE
Create r-cmls recipe

### PR DESCRIPTION
**Pull request by PertuyF**
Create r-cmls recipe using bgruening/conda_r_skeleton_helper.

### CODE REVIEW

**Comment by PertuyF** [comment]
@conda-forge/help-r Should be ready to merge.

File: recipes/r-cmls/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/CMLS_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/CMLS/CMLS_{{ version }}.tar.gz
+  sha256: 765f5d122fb37b93ac121bb41989af6b4260442bc55ac82fcb7a6a4335c68294
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
```
Comment (bgruening) [inline recipes/r-cmls/meta.yaml]: This one should be noarch and the `{{posix}}zip               # [win]` should be removed.

**Comment by PertuyF** [comment]
Where are we:

* with `{{posix}}zip # [win]` the build succeeds for windows, but the linter requires to remove `noarch`. 
* this should be a noarch package, but with `noarch` we can't have a platform selector on the build section.
* without `zip`, tests from the windows build fails.

[`zip`](https://anaconda.org/conda-forge/zip) from conda-forge is not built for windows, but some replacements could used, as [`r-zip`](https://anaconda.org/conda-forge/r-zip), or maybe [`zip-cos6-x86`](https://anaconda.org/anaconda/zip-cos6-x86_64) from anaconda default channel.
Any though @bgruening ?

The same will apply to PRs #7223 and #7224.

**Comment by bgruening** [comment]
For the moment leave it like this. @jdblischak is working on a proper fix imho.

**Pull request merged**